### PR TITLE
Updated GNOME runtime to 3.38

### PR DESCRIPTION
**Pull request by Eonfge**
Also include dependencies and application

### CODE REVIEW

**Comment by Eonfge** [comment]
I also added libcanberra which is required for Evolution Data Server, and I included libhandy which is a soft dependency of Contacts.

**Comment by Eonfge** [comment]
@bilelmoussaoui 
I only can't test the 'make photo' functionality for contacts, because I don't own a webcam. Can you test that?

File: org.gnome.Contacts.yml [hunk 1]
```diff
@@ -119,6 +126,17 @@ modules:
...
 
+  - name: libhandy
+    buildsystem: meson
+    config-opts:
+      - -Dexamples=false
+      - -Dglade_catalog=disabled
+      - -Dtests=false
+    sources:
+      - type: archive
+        url: https://gitlab.gnome.org/GNOME/libhandy/-/archive/1.0.0/libhandy-1.0.0.tar.gz
```
Comment (bilelmoussaoui) [inline org.gnome.Contacts.yml]: Use download.gnome.org whenever possible

Comment (Eonfge) [reply org.gnome.Contacts.yml]: Any specific reason? Are those files cached?

Comment (Eonfge) [reply org.gnome.Contacts.yml]: Done, although I still wonder why

Comment (bilelmoussaoui) [reply org.gnome.Contacts.yml]: Because download.gnome.org has and will always be there :)

**Comment by bilelmoussaoui** [comment]
Thanks!

**Pull request merged**